templates/contacts.json

```diff
@@ -10,6 +10,30 @@
       "system": "{{@ system @}}",
       "type": "rest"
     },
+    "transform": {
+      "rules": {
+        "default": [
+          [
+            "copy",
+            "*"
+          ],
+          ["add", "$last_modified",
+          ["datetime",
+            ["integer",
+              ["*",
```

Comment:
Will probably change this to "multiply" since we use * for all, makes it a little easier to read